invenio_rdm_records/administration/views/oai.py

```diff
@@ -61,9 +61,23 @@ class OaiPmhEditView(AdminResourceEditView):
     list_view_name = "OAI-PMH"
 
     form_fields = {
-        "name": {"order": 2, "text": "Set name"},
-        "spec": {"order": 3, "text": "Set spec"},
-        "search_pattern": {"order": 4, "text": "Search query"},
+        "name": {
+            "order": 2,
+            "text": "Set name",
+            "description": "A short human-readable string naming the set.",
```

Comment:
Should they be i18n? maybe @kpsherva or @jennur knows?

```suggestion
            "description": _("A short human-readable string naming the set."),
```